### PR TITLE
Multichain activity screen

### PR DESCRIPTION
**Pull request by vthomas13**
## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/components/app/transaction-icon/transaction-icon.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
 .transaction-icon {
   &__grey-circle {
-    height: 28px;
-    width: 28px;
-    border-radius: 14px;
+    height: 32px;
+    width: 32px;
+    border-radius: 16px;
     background: var(--color-background-alternative);
```
Comment (NidhiKJha) [inline ui/components/app/transaction-icon/transaction-icon.scss:6]: Let's use [AvatarIcon](https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-avataricon--docs) here. We won't have to write extra css then

File: ui/components/app/transaction-icon/transaction-icon.js [hunk 2]
```diff
@@ -50,10 +60,17 @@ export default function TransactionIcon({ status, category }) {
         `The category prop passed to TransactionIcon is not supported. The prop is: ${category}`,
       ),
     );
-    return <div className="transaction-icon__grey-circle" />;
+    return <Box className="transaction-icon__grey-circle" />;
```
Comment (NidhiKJha) [inline ui/components/app/transaction-icon/transaction-icon.js:63]: Can we use AvatarIcon here as well?

File: ui/components/ui/list-item/index.scss [hunk 3]
```diff
@@ -81,6 +82,14 @@
     }
   }
 
+  &__top-content{
+    @include H7;
+    grid-area: top;
+    color: var(--color-text-alternative);
+    margin-top: 4px;
+    margin-bottom: 16px;
+  }
```
Comment (NidhiKJha) [inline ui/components/ui/list-item/index.scss:91]: Let's use Text for this

File: ui/helpers/utils/util.js [hunk 4]
```diff
@@ -39,8 +39,8 @@ export function formatDate(date, format = "M/d/y 'at' T") {
 
 export function formatDateWithYearContext(
   date,
-  formatThisYear = 'MMM d',
-  fallback = 'MMM d, y',
+  formatThisYear = "MMM d 'at' h:mm a",
+  fallback = "MMM d, y 'at' h:mm a",
```
Comment (NidhiKJha) [inline ui/helpers/utils/util.js:43]: I think we can create another function for this format in that case we would not have to deal with test failure and updating the hook. wdyt?

File: ui/components/ui/list-item/index.scss [hunk 5]
```diff
@@ -12,6 +12,7 @@
   display: grid;
   grid-template-columns: 0fr repeat(11, 1fr);
   grid-template-areas:
+    'top  top      top      top      top      top      top      top      top   top   top   top'
```
Comment (NidhiKJha) [inline ui/components/ui/list-item/index.scss:15]: Following our conversation, I think you can get rid of grid and do it with Flex. We can follow the same pattern as what we have for multichainTokenList

File: ui/components/ui/list-item/list-item.component.js [hunk 6]
```diff
@@ -33,6 +34,9 @@ export default function ListItem({
         }
       }}
     >
+      {topContent ? (
+        <div className="list-item__top-content">{topContent}</div>
```
Comment (NidhiKJha) [inline ui/components/ui/list-item/list-item.component.js:38]: I think we can use Text for topContent here?
Screenshot 2023-06-08 at 12 02 17 AM

**Review by NidhiKJha — CHANGES_REQUESTED** [review]
Really good start. Left some comments :)

**Comment by vthomas13** [comment]
dupe branch

**Pull request closed without merging**